shared/pkg/analyzer/trivy/trivy.go

```diff
@@ -96,6 +97,9 @@ func (a *Analyzer) Run(sourceType utils.SourceType, userInput string) error {
 				Output:       &output,     // Save the output to our local buffer instead of Stdout
 				ListAllPkgs:  true,        // By default Trivy only includes packages with vulnerabilities, for full SBOM set true.
 			},
+			VulnerabilityOptions: trivyFlag.VulnerabilityOptions{
+				VulnType: trivyTypes.VulnTypes, // Scan all vuln types trivy supports
```

Comment:
Could we update the comment here to explain that even though we don't have the vuln security check turned on we need to provide the whole list otherwise trivy disables the analyzer for language packages

Reply:
@sambetts-cisco, yes, done.